sdk/identity/azure-identity/src/samples/Azure-Cache-For-Redis/Jedis/Azure-AAD-Authentication-With-Jedis.md

```diff
@@ -103,7 +103,7 @@ DefaultAzureCredential defaultAzureCredential = new DefaultAzureCredentialBuilde
 
 // Fetch an Azure AD token to be used for authentication. This token will be used as the password.
 // Note: The Scopes parameter will change as the Azure AD Authentication support hits public preview and eventually GA's.
```

Comment:
Now that we're in public preview, this comment should change. Do we expect the scope to change before GA? Is it really a GUID that's used?

Reply:
@samsaha-ms did mention, the team will be looking to add a more user friendly and consistent scope value like KV, Storage etc use (i.e KV -> "https://vault.azure.net/.default")